Client/game_sa/CModelInfoSA.h

```diff
@@ -151,34 +151,53 @@ class CBaseModelInfoSAInterface
 
     unsigned short usDynamicIndex : 16;            // +16
 
     // Flags used by CBaseModelInfo
-    unsigned char bHasBeenPreRendered : 1;            // +18
-    unsigned char bAlphaTransparency : 1;
-    unsigned char bIsLod : 1;
-    unsigned char bDontWriteZBuffer : 1;
-    unsigned char bDontCastShadowsOn : 1;
+    unsigned char bHasBeenPreRendered : 1;  // Confirmed: Has Pre-Rendered          // +18
+    unsigned char bDrawLast : 1;            // Confirmed: Drawn Last
     unsigned char bDrawAdditive : 1;
-    unsigned char bDrawLast : 1;
+    unsigned char bDontWriteZBuffer : 1;    // Confirmed: Don't Write Z Buffer
+    unsigned char bDontCastShadowsOn : 1;   // Confirmed: Don't Cast Shadow On
+    unsigned char bIsLod : 1;               // Confirmed: is LOD
+    unsigned char bAlphaTransparency : 1;
     unsigned char bDoWeOwnTheColModel : 1;
-
-    unsigned char dwUnknownFlag25 : 1;            // +19
-    unsigned char dwUnknownFlag26 : 1;
-    unsigned char dwUnknownFlag27 : 1;
-    unsigned char bSwaysInWind : 1;
-    unsigned char bCollisionWasStreamedWithModel : 1;            // CClumpModelInfo::SetCollisionWasStreamedWithModel(unsigned int)
-    unsigned char bDontCollideWithFlyer : 1;                     // CAtomicModelInfo::SetDontCollideWithFlyer(unsigned int)
-    unsigned char bHasComplexHierarchy : 1;                      // CClumpModelInfo::SetHasComplexHierarchy(unsigned int)
-    unsigned char bWetRoadReflection : 1;                        // CAtomicModelInfo::SetWetRoadReflection(unsigned int)
-
+    union
+    {
+        struct
+        {            // Atomic flags
+            unsigned char bIsRoad : 1;
+            unsigned char : 1;
+            unsigned char bDontCollideWithFlyer : 1;            // CAtomicModelInfo::SetDontCollideWithFlyer(unsigned int)
+            unsigned char nSpecialType : 4;
+            unsigned char bWetRoadReflection : 1;               // CAtomicModelInfo::SetWetRoadReflection(unsigned int)
+        };
+        struct
+        {            // Vehicle flags
+            unsigned char bUsesVehDummy : 1;
+            unsigned char : 1;
+            unsigned char nCarmodId : 5;
+            unsigned char bUseCommonVehicleDictionary : 1;
+        };
+        struct
+        {            // Clump flags
+            unsigned char bHasAnimBlend : 1;                    // Confirmed: has animation
+            unsigned char bHasComplexHierarchy : 1;             // CClumpModelInfo::SetHasComplexHierarchy(unsigned int)
+            unsigned char bUnknown1 : 1;
+            unsigned char bSwaysInWind : 1;                     // Confirmed: Sways in wind
+            unsigned char bCollisionWasStreamedWithModel : 1;   // CClumpModelInfo::SetCollisionWasStreamedWithModel(unsigned int)
+            unsigned char bOwnsCollisionModel : 1;
+            unsigned char bUnknown2 : 1;
+            unsigned char bTagDisabled : 1;
+        };
+    };
+   
```

Comment:
Again, is any of this inside the scope of this PR? This should be submitted separately if it's not essential to fixing the collision bug.